styles2.css

```
@@ -0,0 +1,547 @@
+Указываем box sizing *,
```

is it a comment?
```suggestion
/* Указываем box sizing */
*,
```
and do not use russian language in your code. comments should be in english too